### ISSUE TITLE
Object numbers in example images

### ISSUE THREAD

**Issue by simleo** [CellMigStandOrg/Tracks#16]
The examples in the spec page would be much easier to understand if we placed the numbers from the tables on top of corresponding objects in the pictures. Do we have the image sources (e.g., SVG)?

**Comment by pcmasuzzo** [comment]
Yes, agree. I could work on this. Have the SVG as Google Drawings/slides somewhere, will have a look.
Can we address this in `v0.2`?
